chromium/rules.js

```diff
@@ -493,7 +493,7 @@ RuleSets.prototype = {
 
     var potentiallyApplicable = this.potentiallyApplicableRulesets(hostname);
     for (let ruleset of potentiallyApplicable) {
-      if (ruleset.cookierules !== null && ruleset.active) {
+      if (ruleset.cookierules && ruleset.active) {
```

Comment:
what if `ruleset.cookierules` is falsey but alse `!== null`?

Reply:
It can only be an array or `null` or `undefined`.